gigascience/gigadb-website
# A11y - Home page - hero section - text contrast on image

# Pull request for issue: #1366

This is a pull request for the following functionalities:

* Increase color contrast of hero banner (text on image)

## How to test?

* Visual check of pages:
  * http://gigadb.gigasciencejournal.com:9170/
  * http://gigadb.gigasciencejournal.com:9170/search/new?keyword=Software&type%5B%5D=dataset&dataset_type%5B%5D=Software
* If you want to be more accurate you might use for example this page https://colourcontrast.cc/ to compare the lightest color you see in the banner image (close to this #646464) with the overlaying text color (#ffffff) and confirm that the AA Normal criterion is fulfilled

## How have functionalities been implemented?

Original:

![image](https://github.com/gigascience/gigadb-website/assets/143437854/939a726f-aa4c-4a8d-95e6-c3be5a313b5b)

New:

![image](https://github.com/gigascience/gigadb-website/assets/143437854/9a64929b-0ee0-4bf7-aeee-0d2cee8eb16c)

* Font size slightly increased
* Background darkened with a semiopaque overlay
* Darken background color of banner so that if image errors out text is still readable, i.e.

![image](https://github.com/gigascience/gigadb-website/assets/143437854/ce0033d8-acf3-41e9-bd23-c87c6b7cd611)

## Any issues with implementation?

None

## Any changes to automated tests?

None

## Any changes to documentation?

None

## Any technical debt repayment?

None

## Any improvements to CI/CD pipeline?

None
